recipe/meta.yaml

```diff
@@ -44,3 +44,5 @@ extra:
     - scopatz
     - gonuke
     - bam241
+    - katyhuff
+    - FlanFlanagan
```

Comment:
@katyhuff @FlanFlanagan are you OK with being listed as maintainers here?

Reply:
yes